ogusa/aggregates.py

```diff
@@ -303,10 +307,14 @@ def revenue(r, w, b, n, bq, c, Y, L, K, factor, theta, etr_params,
             np.squeeze(p.lambdas) *
             np.tile(np.reshape(p.omega[:p.T, :], (p.T, p.S, 1)),
                     (1, 1, p.J)))
+        pop_weights_ubi = (
```

Comment:
@rickecon Why not use the `pop_weights` array above?  `pop_weights_ubi` seems to be the same, just a different shape.  Probably better to have the `ubi` array conform to the same dimensions used for other TxSxJ arrays.

Reply:
Ah, this was the easiest way for me to reshape the `pop_weights` array. The `pop_weights` and `pop_weights_ubi` contain the same information, but they are different dimensions. `pop_weights` is dimension (T, S, J). `pop_weights_ubi` is dimension (S, J, T) to be conformable with the `ubi` array. You see in line 317 that I have to multiply these two 3D arrays before summing over the first two dimensions. @jdebacker

Reply:
Right, but my question is - why is the `ubi` array SxJxT, whenI think all other arrays are TxSxJ?  Seems like breaking from the norm can cause confusion (and duplicating objects as done here).

Reply:
OK. I'll change the household `ubi` array to be (T, S, J).